libgnucash/engine/mocks/fake-qofquery.h

```diff
@@ -0,0 +1,51 @@
+/**
+ * @file fake-qofquery.h
+ *
+ * @brief Mocking qof queries
+*/
+
+#ifndef FAKE_QOFQUERY_H
+#define FAKE_QOFQUERY_H
+
+#include <gmock/gmock.h>
+
+#include <qofquery.h>
+
+extern "C"
+{
+#include <Query.h>
+}
+
+/** Fake object providing functionality similar to \c QofQuery
+ *
+ * @note \c QofQuery is a \c typedef for \c struct _QofQuery, which is not
+ * public. Therefore class QofFakeQuery is not derived from \c QofQuery.
```

Comment:
`@c` instead of `\c`, and note that `@c struct _QofQuery` puts only `struct` in tt-face; to get both in use either `<tt>struct _QofQuery</tt>` or `@code struct _QofQuery @endcode`. But be consistent: This is the only place you marked words for tt-face.

Reply:
> `@c` instead of `\c`

Done

> and note that `@c struct _QofQuery` puts only `struct` in tt-face [...] But be consistent: This is the only place you marked words for tt-face

The point is, I tried to mark up only names like `QofQuery`, which are not linked to any doxygen generated documentation. Therefore I used tt-face for `QofQuery` but not for `_QofQuery` and `QofFakeQuery` for instance. But this is actually problematic, since missing doxygen documentation may be only a temporary state or a question of doxygen configuration. Therefore I removed most of the `@c` and kept it only for `typedef` and `struct`.

Or should I completely remove all `@c`?